recipes/ott-jax/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ott-jax" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ott-jax-{{ version }}.tar.gz
+  sha256: adcd3c59ac9223a962b7a9244c4f53cc9857bf2461530583edb1ea8a1fc37ea3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.7
+    - jax >=0.1.67
+    - jaxlib >=0.1.47
+    - numpy >=1.18.4,!=1.23.0
+    - matplotlib-base >=3.0.0
+    - flax >=0.5.2
+    - optax >=0.1.1
+    - scipy >=1.7.0
+    - importlib-metadata >=1.0
+    - typing-extensions
```

Comment:
```suggestion
    - absl-py >=0.7.0
    - jax >=0.1.67
    - jaxlib >=0.1.47
    - numpy >=1.18.4,!=1.23.0
    - matplotlib-base >=2.0.1
    - flax >=0.3.6
    - optax >=0.1.1
    - scipy >=1.7.0
    - scikit-learn >=1.0
    - importlib-metadata >=1.0
    - typing-extensions
```

Reply:
I removed the `absl-py/scikit-learn`, since they were both used only during testing (unfortunately, they were still present in the previous release on PyPI); this is also what prevents currently from having the `pip check`, as suggested in https://github.com/conda-forge/staged-recipes/pull/21408#discussion_r1038303722.
Imho it's better to not include them here rather than making a new release with minor changes, but don't have strong opinion.

Reply:
Please add a comment about that and comment out the pip-check so  we can enable them when there is a new release.

Reply:
Done.